pkgs/development/python-modules/asyncua/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, aiosqlite
+, aiofiles
+, pytz
+, python-dateutil
+, sortedcontainers
+, cryptography
+, typing-extensions
+, pytestCheckHook
+, pytest-asyncio
+, pytest-cov
+, pytest-mock
+, coverage
+, asynctest
+}:
+
+buildPythonPackage rec {
+  pname = "asyncua";
+  version = "0.9.98";
+
+  src = fetchFromGitHub {
+    owner = "FreeOpcUa";
+    repo = "opcua-asyncio";
+    rev = "v${version}";
+    sha256 = "sha256-nxpnIJcv/qXMhJJdG/DVrg8pWkMBGvh3V/2n+jjOz/Q=";
+  };
+
+  propagatedBuildInputs = [
+    aiosqlite
+    aiofiles
+    pytz
+    python-dateutil
+    sortedcontainers
+    cryptography
+    typing-extensions
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    pytest-asyncio
+    pytest-cov
+    pytest-mock
+    coverage
+    asynctest
```

Please remove all coverage -related parts. This doesn't add value for us.

```suggestion
    pytest-mock
    asynctest
```

It could be required to remove `--cov` and alike with `substituteInPlace` in the `postPatch` phase.